Sources/Purchasing/Offering.swift

```diff
@@ -178,6 +179,25 @@ import Foundation
 
     // swiftlint:disable cyclomatic_complexity
 
+    /// Initialize an ``Offering`` given a list of ``Package``s.
+    @objc
+    public convenience init(
```

Comment:
I wonder, should we deprecate these constructors? Not sure if we want to push devs into just a single constructor... Maybe we should even try to make the "main" constructor internal to avoid these issues but we would need to keep the current ones public for backwards compatibility